api/server/server.go

```diff
@@ -554,7 +554,7 @@ func (s *Server) getContainersStats(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	return s.daemon.ContainerStats(vars["name"], boolValue(r, "stream"), ioutils.NewWriteFlusher(w))
+	return s.daemon.ContainerStats(vars["name"], boolValueOrDefault(r, "stream", true), ioutils.NewWriteFlusher(w))
```

Comment:
Actually I know I suggested the default value thing... but now I'm thinking `stream := r.Form.Get("stream") == "" || boolValue(r, "stream")`